t/cobrand/zurich.t

```diff
@@ -345,6 +346,27 @@ subtest "Auto select private category and input description from url" => sub {
     $mech->clear_emails_ok;
 };
 
+subtest "Auto select category and input description from front page" => sub {
+    my $module = Test::MockModule->new('FixMyStreet::Geocode::Zurich');
+    $module->mock(string => sub { { latitude => 47.376056, longitude => 8.525481 } });
+
+    $mech->get_ok('/?prefill_category=Allgemein&prefill_description=Test 5678');
+    $mech->submit_form_ok({ with_fields => {
+        pc  => 'Langstrasse',
+    }});
+    is $mech->uri->path, '/around', 'Redirected to around page';
```

Comment:
It is, but could we just check the prefill desc is within the source of this page, and the category list isn't being shown, just so it isn't lost in future by some accident.